overlays/haskell.nix

```diff
@@ -252,22 +252,22 @@ final: prev: {
         };
 
         update-index-state-hashes = import ../scripts/update-index-state-hashes.nix {
-            inherit (final.haskell-nix) indexStateHashesPath nix-tools;
+            inherit (final.haskell-nix) indexStateHashesPath internal-nix-tools;
```

Comment:
Extreme nit but I'd probably leave the parameter called `nix-tools` and do `nix-tools = internal-nix-tools`.